### ISSUE TITLE
deb-get broadcasts spam information about tonelib deb

### ISSUE THREAD

**Issue by kupiqu** [wimpysworld/deb-get#962]
First of all, thank you for your work!

Some context: I added deb-get update to my bash alias for updating deb packages in the system, which I use routinely and since few days ago I see this message:

```
[!] ERROR! Missing required information of website package tonelib-noisereducer:
URL=https://tonelib.net/download/221222/ToneLib-NoiseReducer-amd64.deb
VERSION_PUBLISHED=
```

At first, I didn't know if my system was compromised or what. After a bit of research I saw this is related to deb-get's tonelib repo, which I never heard about before.

I think it would be great if we are not bothered by this noise (imagine that several repos start spreading error information; it doesn't look good). A good option would be that deb-get internally filters warning/error information according to what is already instaled for the case of updates (this way we would not be bothered about issues with the tonelib repo we're not interested about), and according to what is pretended to be installed (if I try to install tonelib in my system, then I want to see this information that sth is wrong with it).

Would that make sense? Thanks

**Comment by philclifford** [comment]
It would be nice if you were not inconvenienced by this error message.  Giving the error only to users with the app installed would be ideal, except that would require maintainers to have everything supported installed (not trivially possible as some packages conflict with or replace others) in order to notice and try and attend to such issues if they are more than transient glitches. 

A possible solution would be to add some sort of "verbose" flag and silence these errors except where the package is installed OR the update is requested to be "verbose".

**Comment by miljanuscumlic** [comment]
I just want to note that this is not just a noise related issue. After this error happens, none of the files that comes after tonelib-noisereducer.html are being updated.

**Comment by KlfJoat** [comment]
I, also, am receiving this error on two hosts, and cannot install Zoom because of it on either of my hosts.

My `deb-get list` ends at 'tonelib-metal', the item immediately preceding 'tonelib-noisereducer'.

There's a PR to fix this specific issue, but it's 3 weeks old and has yet to be merged, leading every `deb-get` user who has performed `deb-get update` in the last 3 weeks to have an error.

**Every. Single. User.**

This isn't a "[requiring] maintainers to have everything supported installed… in order to notice and try and attend to such issues" situation like @philclifford suggests. It is a problem with the repo, one which can likely be tested for with a daily GitHub Action by running a specialized test OR just running `deb-get update`, then auto-opening a ticket upon failure.

However, that ignores the most important question:

### How is `deb-get` not resilient against this?!?!?

By necessity it works off of external data. So how is there no handling for empty or garbled return data? Why does it fail hard instead of gracefully? Why can't it throw a Warning and skip 'tonelib-noisereducer', then continue?

**Comment by Daeraxa** [comment]
This isn't a new one either, I remember this happening before, I think with Jellyfin which I logged some months ago - https://github.com/wimpysworld/deb-get/issues/822

**Comment by KlfJoat** [comment]
@philclifford @flexiondotorg 

This is still happening?

Seriously, what's going on? How have so many other repository items been added or updated, but this one remains unfixed?

Users have been prevented from installing 36 different programs with `deb-get` for over a month and a half because of this.

Why hasn't #964 been merged?

Why hasn't `deb-get` itself been updated to be resilient against this entirely foreseeable and repeated (#822) problem?

**Comment by fedz1** [comment]
I am having the same issue and I was wondering is there a way to temporarily exclude some packages from being checked? I ended up deleting the html/json that cause the error, but it is not a very elegant solution. Thank you

**Comment by philclifford** [comment]
> ... is there a way to temporarily exclude some packages from being checked? 

Not currently.  You could drop the unmerged fixed package definition from #963 (or create your own for any future breakage and consider submitting a PR) into the local override folder, e.g. `/etc/deb-get/99-local.d/tonelib-noisereducer`.
Alternatively you could 

```
echo "https://raw.githubusercontent.com/philclifford/deb-get/extras/91-fixes" | sudo tee /etc/deb-get/91-fixes.repo 
```

and after `deb-get update` you should have all the unmerged fixes to package definitions as overrides.

**Comment by mralusw** [comment]
If I drop a package definition into 99-local.d, can I rename it to `tonelib-noisecreator`, and also make it so that it doesn't consume any bandwidth / time / GH token usage at all?
